pymatgen/core/tests/test_composition.py

```diff
@@ -395,6 +401,10 @@ def test_equals(self):
         )
```

Comment:
@shyuep While updating the tests, I found this odd `random.randint` code in `CompositionTest.test_equals()`. I don't understand what it's doing.

Couldn't that whole block be replaced by this?

```py
        c1, c2 = self.comp[:2]
        self.assertTrue(c1 == c1)
        self.assertFalse(c1 == c2)
```

Reply:
I see. I'll add a comment for future readers.